plugin/sampling/strategystore/static/fixtures/probablistic_with_default_operation_strategy.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I would recommend naming the file after the test name and some other dimension like serviceA.